File: C/somavetor.c
// Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida: imprimir todos os elementos do vetor; mostrar na tela a soma e a média dos elementos do vetor.
#include <stdio.h>

int main(){
	int n;
	int i;
	float numeros[100];
	float soma;
	float media;
	
	printf("Quantos numeros voce vai digitar?: ");
	scanf("%d", &n);
	
	for(i = 0; i < n; i++){
		printf("Digite um numero: ");
		scanf("%f", &numeros[i]);
	}
	
	printf("Valores: ");
	for(i = 0; i < n; i++){
			printf("%.2f\n", numeros[i]);
		}
	
	for(i = 0; i < n; i++){
		soma = soma + numeros[i];
	}
	printf("Soma: %.2f\n", soma);
	
	for(i = 0; i < n; i++){
		media = soma / n;
	}
	printf("Media: %.2f\n", media);
}

File: P/soma_vetor.py
# Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida: Imprimir todos os elementos do vetor e mostrar na tela a soma e a média dos elementos do vetor

soma = 0

n = int(input("Quantos numeros voce vai digitar? "))
vet = [ 0 for x in range (n)]

for i in range (0, n):
    vet[i] = int(input("Digite um numero: "))

for i in range (0, n):
    soma = soma + vet[i]

media = soma / n 

print(f"Valores: ")
for i in range(0, n):
    print(vet[i])

print(f"Soma: {soma}")
print(f"Media: {media}")

